src/playground/index.html

```diff
@@ -44,10 +44,16 @@
         direction: 90
     });
     var drawableID2 = renderer.createDrawable();
-    renderer.updateDrawableProperties(drawableID2, {
-        skin: 'https://cdn.assets.scratch.mit.edu/internalapi/asset/' +
-            '09dc888b0b7df19f70d81588ae73420e.svg/get/'
-    });
+    var image = new Image();
+    image.crossOrigin = 'anonymous';
+    image.src = 'https://cdn.assets.scratch.mit.edu/internalapi/asset/' +
+        '09dc888b0b7df19f70d81588ae73420e.svg/get/';
```

Comment:
Just to be safe, please attach the `onload` function before setting `image.src`. It may work as you have it, depending on the browser and cache state, but setting `onload` first is more reliable.
See also: https://stackoverflow.com/questions/14648598/is-it-necessary-to-set-onload-function-before-setting-src-for-an-image-object